test/Microsoft.DotNet.ToolPackage.Tests/Microsoft.DotNet.ToolPackage.Tests.csproj

```diff
@@ -52,5 +57,7 @@
     </MSBuild>
     <MSBuild BuildInParallel="False" Projects="SampleGlobalTool/consoledemo.csproj" Targets="pack" Properties="Configuration=Release;NuspecFile=includepublish.nuspec;NuspecBasePath=$(testAssetSourceRoot);PackageOutputPath=$(OutputPath)/TestAssetLocalNugetFeed">
     </MSBuild>
+    <Copy SourceFiles="ExtraTestFeedPackages/microsoft.netcore.platforms.2.0.1.nupkg" DestinationFolder="$(OutputPath)/TestAssetLocalNugetFeed" />
```

Comment:
Instead of explicitly copying this here, I'd suggest adding the file to the `Content` item.

Reply:
Fixed, use Content